packages/newspringchurchapp/src/content-single/Features/TextFeature.js

```diff
@@ -13,10 +13,11 @@ const TextFeature = ({
   card,
 }) => {
   const [isPressed, press] = useState(false);
-  const bodyWithBlank = body.replace(/__(.*)__/gm, (match, p1) =>
+  const bodyWithComma = body.replace(/%2C/, ',');
```

Comment:
...in core

Reply:
Would we not just do the same thing in the backend? If so, does it really matter where we do this? I'm of the opinion that it doesn't.

Reply:
it's a bug in core that the message data field of a text feature returns text that isn't parsed correctly, that should be fixed.

Reply:
we would have to repeat this code everywhere on the front we used a text feature in the future. technically this same thing is broken in core on the front end right now, why would we not fix it there?

Reply:
I guess I had the assumption that the text feature wouldn't be included outside of a WeekendContentItem? I'm honestly not sure. We can PR this against core though.

Reply:
So yeah, a `TextFeature` could technically be used in multiple places, not just the `WeekendContentItem`. I think I agree that this should probably be fixed in core. And I guess I want to know if there is a way to parse the text so that you don't have to do `bodyWithComma` and `bodyWithQuote` and `bodyWith...`, you get the idea. But that can be figured out as this task gets taken on. Parsing all of the text at once would be ideal.

Reply:
I'm making a PR against Core for this to be handled on the backend. I think we should leave this in until that gets merged.

Reply:
I disagree, we likely won't even have notes before it gets merged and even if we do, just make sure there are no commas

Reply:
Are we sure that it's only commas that are at issue here? Are there other punctuation marks or the like that would/could/are causing problems?

Reply:
Commas are all I'm aware but I'm sure there's more. That's why this particular fix really doesn't serve us very well.

Reply:
Yeah. I'm not sure off the top of my head how to do it, but whatever the fix is should parse all the special characters correctly.

Reply:
@cdanwards I think I'm in agreement with @redreceipt here. Let's take out the comma fix (since I really feel like the fix should be wider ranging than just a comma) and we can then merge this in. I want to make sure we get this in our new beta release tomorrow. We'll make sure on our end that the notes we do have don't include anything breaking until the core fix is done and in our codebase. Feel free to make an issue in core and then fix that issue. Go team, go!